core-services/prow/02_config/openshift/cluster-monitoring-operator/_prowconfig.yaml

```diff
@@ -68,6 +68,25 @@ tide:
     - needs-rebase
     repos:
     - openshift/cluster-monitoring-operator
+  - includedBranches:
+    - main
+    - master
+    labels:
+    - approved
+    - docs-approved
+    - lgtm
+    - px-approved
```

Comment:
Not sure what `px-approved` refers to, but I've never seen this label before in CMO so I don't think we should require it.

Reply:
Unfortunately we cannot support exceptions at this time.

Reply:
px(product experience) is what we used to call CEE (the support org).  

For any feature going in, we need PX to ack that they have received(or will be receiving) whatever training they feel they need to support the feature.